build-system/json-schemas/caches.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "List of known AMP caches",
```

Comment:
The check in this file is far more comprehensive than the one in `build-system/tasks/caches-json.js`. Can we delete the `caches-json` task and its associated code?

https://github.com/ampproject/amphtml/blob/6b53dc6b550ba2e6b2beb5ae4cca938c7ebe0669/build-system/pr-check/build-targets.js#L153-L161
https://github.com/ampproject/amphtml/blob/6b53dc6b550ba2e6b2beb5ae4cca938c7ebe0669/build-system/pr-check/checks.js#L79-L81
https://github.com/ampproject/amphtml/blob/6b53dc6b550ba2e6b2beb5ae4cca938c7ebe0669/build-system/tasks/pr-check.js#L74-L76

Reply:
Done.